src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -329,13 +329,17 @@ protected function fallbackName($name)
     /**
      * Write the contents of a file.
      *
-     * @param  string  $path
-     * @param  \Psr\Http\Message\StreamInterface|\Illuminate\Http\File|\Illuminate\Http\UploadedFile|string|resource  $contents
+     * @param  \Psr\Http\Message\StreamInterface|\Illuminate\Http\File|\Illuminate\Http\UploadedFile|string|resource|string  $path
+     * @param  \Psr\Http\Message\StreamInterface|\Illuminate\Http\File|\Illuminate\Http\UploadedFile|string|resource|mixed  $contents
```

Comment:
`mixed` can never be combined with another type.

Reply:
Sorry about that - fixed.